.github/workflows/c1-v2.yml

```
@@ -0,0 +1,179 @@
+          aws-region: ap-northeast-2
+        
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v2
+
+      - name: Build V2 Docker image
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: lifebookshelf-server-v2
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:48: Double quote to prevent globbing and word splitting

(shellcheck)